cfme/fixtures/v2v_fixtures.py

```diff
@@ -212,21 +212,35 @@ def vddk_url():
     return url
 
 
-def get_conversion_data(target_provider):
+def get_conversion_data(appliance, target_provider):
     if target_provider.one_of(RHEVMProvider):
-        resource_type = "ManageIQ::Providers::Redhat::InfraManager::Host"
-        engine_key = conf.credentials[target_provider.data["ssh_creds"]]
-        auth_user = engine_key.username
-        ssh_client = ssh.SSHClient(
-            hostname=target_provider.hostname,
-            username=engine_key.username,
-            password=engine_key.password,
-        )
-        private_key = ssh_client.run_command("cat /etc/pki/ovirt-engine/keys/engine_id_rsa").output
-        try:
-            hosts = [h.name for h in target_provider.hosts.all()]
-        except KeyError:
-            pytest.skip("No conversion host on provider")
+        if appliance.version < '5.11':
+            resource_type = "ManageIQ::Providers::Redhat::InfraManager::Host"
+            engine_key = conf.credentials[target_provider.data["ssh_creds"]]
+            auth_user = engine_key.username
+            ssh_client = ssh.SSHClient(
+                hostname=target_provider.hostname,
+                username=engine_key.username,
+                password=engine_key.password,
+            )
+            private_key = ssh_client.run_command(
+                "cat /etc/pki/ovirt-engine/keys/engine_id_rsa").output
+            try:
+                hosts = [h.name for h in target_provider.hosts.all()]
+            except KeyError:
+                pytest.skip("No conversion host on provider")
+
+        elif appliance.version > '5.10':
```

Comment:
Optional : since we are testing for only two versions now , you can just do `else`.